### PR TITLE
BARSIGNS-BARSIGNS-BARSIGNS!

### PR DESCRIPTION
**Pull request by Zargoar**
## About The Pull Request

We need the last of the signs added to the game so everyone can see and use them.
Adds the Red Mons, The Rune, The Wizard and MOTHS-MOTHS-MOTHS bar signs that were created during the bar sign contest. 

## Why It's Good For The Game

More options for the bar name is a good thing for player choice and diverse player stories.
Giving our bartenders more options helps player agency.

## Changelog

the-red-mons

the-rune

the-wizard

moths-moths-moths

:cl:
add: Adds in the Red Mons, The Wizard, The Rune and MOTHS-MOTHS-MOTHS bar signs from the bar sign contest.
Congratulations to the winners!
image: Adds in the Red Mons, The Wizard, The Rune and MOTHS-MOTHS-MOTHS bar signs from the bar sign contest.
/:cl:

### CODE REVIEW

File: code/game/machinery/barsigns.dm [hunk 1]
```diff
@@ -478,6 +478,30 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/barsign, 32)
 	desc = "First stop out of hell, last stop before heaven."
 	neon_color = COLOR_RED
 
+/datum/barsign/the-red-mons
```
Comment (LemonInTheDark) [inline code/game/machinery/barsigns.dm:481]: do not use - in paths. use _

File: code/game/machinery/barsigns.dm [hunk 2]
```diff
@@ -478,6 +478,30 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/barsign, 32)
...
+	name = "The Red Mons"
+	icon_state = "the-red-mons"
+	desc = "Drinks from the Red Planet."
+	neon_color = COLOR_RED
+
+/datum/barsign/therune
```
Comment (LemonInTheDark) [inline code/game/machinery/barsigns.dm:487]: _ here, be consistent

File: code/game/machinery/barsigns.dm [hunk 3]
```diff
@@ -478,6 +478,30 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/barsign, 32)
...
+	name = "The Wizard"
+	icon_state = "the-wizard"
+	desc = "Magical mixes."
+	neon_color = COLOR_RED
+
+	/datum/barsign/months-moths-moths
+	name = "Moths Moths Moths"
+	icon_state = "moths-moths-moths"
+	desc = "LIVE MOTHS!"
+	neon_color = COLOR_RED
```
Comment (LemonInTheDark) [inline code/game/machinery/barsigns.dm:503]: Bro what. Is this tested?

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

File: code/game/machinery/barsigns.dm (hunk 3 above)
Comment (LemonInTheDark) [reply code/game/machinery/barsigns.dm:503]: Regardless this needs to be de-indented

**Comment by LemonInTheDark** [comment]
This getting in depends on the opinion of the spritetainers, can't recall if there was a cut list or not.
Also please credit the authors of these sprites in the changelog

**Comment by Wallemations** [comment]
Show ALL the signs in the PR

**Comment by Ghommie** [comment]
Every new bar sign is a few more kilobytes of resource that each and every player has to download.

**Pull request merged**